MapboxNavigation/RouteMapViewController.swift

```diff
@@ -15,6 +15,26 @@ class RouteMapViewController: UIViewController {
     var topBannerContainerView: BannerContainerView { return navigationView.topBannerContainerView }
     var bottomBannerContainerView: BannerContainerView { return navigationView.bottomBannerContainerView }
     
+    typealias FloatingButtonsPositions = NavigationView.FloatingButtonsPositions
```

Comment:
This type alias won’t be necessary if we declare the enumeration at the top level.